Tentacle/Client.swift

```diff
@@ -390,6 +393,54 @@ public final class Client {
                     .concat(response.links["next"] == nil ? SignalProducer.empty : self.fetchMany(endpoint, page: nextPage, pageSize: pageSize))
             }
     }
+
+    internal func send<Request: RequestType>(_ request: Request, to endpoint: Endpoint, method: String) -> SignalProducer<(Response, Request.Response), Error> where Request.Response == Request.Response.DecodedType {
```

Comment:
That's something that really makes me wonder in FoundationKit. How come it's not an enum already? 🤔